roles/kubernetes/cni/calico/tasks/main.yaml

```diff
@@ -6,7 +6,7 @@
 #
 - name: Install Operator
   ansible.builtin.shell: |
-    kubectl apply -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
+    kubectl create -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
   tags:
     - install
     - update
```

Comment:
Need to remove the `update` target and create an update target just like the `install` with `kubectl replace` to accommodate what `kubectl apply` does. There is also a not in the operator for Calico that `kubectl create` must be used given the data content size. Here's a link to the change and the comment in the code base about this one: 

https://github.com/tigera/operator/blame/master/Makefile#L317-L320

Reply:
I am going to mark these as resolved. Right now the code base is not re-entrant with update so one has to run install then uninstall using the ./oss-k8s.sh script. We'll address the need for `replace` in the near future.